.github/workflows/analysis.yml

```diff
@@ -0,0 +1,24 @@
+name: "Static Analysis"
+on: ["push", "pull_request"]
+jobs:
+  Static-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Static Code Analysis
+        uses: dominikh/staticcheck-action@v1.1.0
+        with:
+          version: "2021.1.2"
```

Comment:
Can we use the latest?
```suggestion
```

Reply:
`Because new versions of Staticcheck introduce new checks that may break your build, it is recommended to pin to a specific version and to update Staticheck consciously.`
They recommend otherwise, but we can try latest and pin a version if it breaks too much.